containers/init/build/php7/Dockerfile

```diff
@@ -30,12 +30,10 @@ RUN gpg2 --keyserver keyserver.ubuntu.com --recv-keys 409B6B1796C275462A17031138
 RUN \curl -sSL https://get.rvm.io | bash -s stable
 
 # Install Ruby 2.5
-RUN apt-get update && apt-get install -y procps && apt-get clean
+RUN apt-get --allow-releaseinfo-change update && apt-get install -y procps && apt-get clean
```

Comment:
Wondering why we want to use this option, is there an update error? Would this option install things with an unverified version?

https://manpages.debian.org/unstable/apt/apt-get.8.en.html
https://manpages.debian.org/unstable/apt/apt-secure.8.en.html

Reply:
I add the `--allow-releaseinfo-change` mainly b/c some of the dependencies have new releases. Several error might pop out if we don't add this, b/c cache are used in previous steps. I won't worry too much on unverified version. Normally, when we use apt-get install, we would install the latest version.